Software/Development/SemesterProject/System/scheduler.c

```diff
@@ -68,6 +74,11 @@ void runTask(void (*functionPTR)())
 /*
  * Tasks definition
  */
+void task100us(void)
+{//running at switching 10kHz (update with switching frequency);
+    setFrequencyFromPot();
```

Comment:
validate with a digital output, to see that the ciclicity is correct.

Reply:
Ciclicity seems correct exept for some glittering which may be due to incorrect triggering.